### DISCUSSION TITLE
想修改一下模板的标题格式该怎么办呀？

### DISCUSSION THREAD

**Discussion by tsiayigong** [stone-zeng/fduthesis#340] [Q&A] [answered]
请问一下，想把模板的标题格式改成：一级章节标题，小二号，黑体，加粗，段前12磅，段后3磅，单倍行距，题序与标题间空一个汉字空格或2个半角空格。二级标题为小三号，加粗，段前18磅，段后12磅，1.5倍行距。题序与标题间空1个空格。三级标题为黑体四号，段前12磅，段后9磅，多倍行距1.25。题序与标题间空1个空格。
但是我按照网上讲的`LaTeX`的`1bp`与`Word`DE `1磅`基本一致，把格式改为了：
```TeX
...
\__my_set_cjk_font_hei:n { SimHei   }
...
\keys_set:nn { ctex }
  {
    chapter =
      {
        format      = \zihao{-2} \bfseries \my@hei \centering,
        beforeskip  = 12 bp,
        afterskip   = 3 bp,
        number      = \__my_arabic:n { chapter },
        fixskip     = true
      },
    section =
      {
        format      = \zihao{-3} \bfseries \my@hei \raggedright,
        beforeskip  = 18 bp,
        afterskip   = 12 bp,
        fixskip     = true
      },
    subsection =
      {
        format      = \zihao{4} \my@hei \raggedright,
        beforeskip  = 12 bp,
        afterskip   = 9 bp,
        fixskip     = true
      }
  }
...
```
得到的结果与`word`给出的误差较大，请问是哪里有问题呀？具体效果如[这里所示](https://ask.latexstudio.net/ask/question/17064.html)
希望能得到大佬的解答

**Comment by SwitWu** [comment, marked as answer]
一级标题段后 3 磅，也就是 3bp，根据换算关系，3bp = 3*25.4/72mm 约等于 1mm，所以截图中 LaTeX 给出的结果是符合预期的。

Word 中标题与正文的间距还和正文的行间距（行距倍数）设置有关，设置得越大，标题与正文的距离越大。虽然给的要求是段后 3 磅，但是很明显截图中的间距不止 3 磅。

在设置 `afterskip` 的时候，你得把下面红色框的高度也考虑进去：

<img width="464" alt="image" src="https://github.com/stone-zeng/fduthesis/assets/39337667/314ff7b2-7e46-4433-a553-637bdacdfbff">

**Reply by tsiayigong** [reply]
这里看起来好像没有包含正文的行距：
![image](https://github.com/stone-zeng/fduthesis/assets/63224494/8c1f31d2-e005-4c42-a21c-87b713328d04)
![image](https://github.com/stone-zeng/fduthesis/assets/63224494/46168b16-3eea-4a0a-94f0-4b2fb901452d)
是因为文档网格的问题吗？

**Reply by SwitWu** [reply]
要我说的话，你多调试几次不就好了

**Reply by tsiayigong** [reply]
好滴，谢谢大佬
